server/src/main/java/io/druid/server/coordination/ZkCoordinator.java

```diff
@@ -86,8 +88,10 @@ public void loadLocalCache()
     }
```

Comment:
Should this log before returning? it seems like it would be very confusing about why things didn't progress.